pos_gift_ticket/__openerp__.py

```diff
@@ -0,0 +1,39 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright :
+#        (c) 2015 FactorLibre S.L. (http://www.factorlibre.com)
+#                 Ismael Calvo <ismael.calvo@factorlibre.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    "name": "Gift Ticket",
+    "version": "0.1",
+    "author": "FactorLibre",
```

Comment:
Could you add OCA ?
Sample : https://github.com/OCA/pos/blob/8.0/pos_pricelist/__openerp__.py#L24
(It's a convention, to make easier search).